pyomo/contrib/mindtpy/initialization.py

```diff
@@ -102,7 +121,7 @@ def init_rNLP(solve_data, config):
                                  solve_data.mip.MindtPy_utils.variable_list,
                                  config, ignore_integrality=True)
             add_oa_cuts(solve_data.mip, dual_values, solve_data, config)
-            # TODO check if value of the binary or integer varibles is 0/1 or integer value.
+            # TODO check if value of the binary or integer variables is 0/1 or integer value.
```

Comment:
I'd remove this comment since the check of whether the variable is an integer is done 2 lines below.